recipes/tree-sitter/all/test_package/conanfile.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build_requirements(self):
+        self.build_requires("cmake/3.17.2")
```

Comment:
CCI assumes this is available so it can be removed here

Reply:
Done! I saw post about cmake, it tells that cmake in required is not forbidden so I decide to add it)